packages/admin/admin/src/form/FinalFormFileSelect.tsx

```diff
@@ -1,393 +1,121 @@
-                                <FormattedMessage id="comet.finalFormFileSelect.dropfiles" defaultMessage="Drop files here to upload" />
-                            </DroppableAreaCaption>
-                        </DroppableArea>
-                    )}
-                    {!disableSelectFileButton && (
-                        <SelectButton
-                            {...slotProps?.selectButton}
-                            disabled={dropzoneDisabled}
-                            variant="contained"
-                            color="secondary"
-                            startIcon={selectIcon}
-                        >
-                            <FormattedMessage id="comet.finalFormFileSelect.selectfile" defaultMessage="Select file" />
-                        </SelectButton>
-                    )}
-                </Dropzone>
-            )}
-            {acceptedFiles.length > 0 && (
-                <FileList {...slotProps?.fileList}>
-                    {acceptedFiles.map((file, index) => (
-                        <FileListItem {...slotProps?.fileListItem} key={index}>
-                            <Tooltip trigger="hover" title={file.name}>
-                                <FileListText {...slotProps?.fileListText}>{file.name}</FileListText>
-                            </Tooltip>
-                            <FileListItemInfos {...slotProps?.fileListItemInfos}>
-                                <Chip label={<PrettyBytes value={file.size} />} />
-                                <IconButton onClick={removeFile(file)}>{deleteIcon}</IconButton>
-                            </FileListItemInfos>
-                        </FileListItem>
-                    ))}
-                </FileList>
-            )}
-            {fileRejections.length > 0 && <FileList>{rejectedFiles}</FileList>}
-            {(fileRejections.length > 0 || isDragReject) && (
-                <ErrorMessage {...slotProps?.errorMessage}>
-                    {errorIcon}
-                    <FormattedMessage id="comet.finalFormFileSelect.errors.unknownError" defaultMessage="Something went wrong." />
-                </ErrorMessage>
-            )}
-            <FormHelperText sx={{ margin: 0 }}>
-                <FormattedMessage
-                    id="comet.finalFormFileSelect.maximumFileSize"
-                    defaultMessage="Maximum file size {fileSize}"
-                    values={{
-                        fileSize: <PrettyBytes value={maxSize} />,
-                    }}
-                />
-            </FormHelperText>
-        </Root>
+                ) : undefined
+            }
+            {...restProps}
+        />
     );
 }
 
 declare module "@mui/material/styles" {
-    interface ComponentNameToClassKey {
-        CometAdminFinalFormFileSelect: FinalFormFileSelectClassKey;
-    }
-
     interface ComponentsPropsList {
         CometAdminFinalFormFileSelect: FinalFormFileSelectProps;
     }
```

Comment:
Do you still need these styles overrides?

Reply:
Since the only thing that could be styled is the `FileSelect`, styling the `FinalFormFileSelect` probably doesn't make sense, as this should look the same as all other `FileSelect` instances. 

Or did you mean the `defaultProps`?
That's probably not necessary either. 
I'll remove them to be consistent with other final-form wrapper components.

Reply:
I meant the theme augmentation in general.

> I'll remove them to be consistent with other final-form wrapper components.

👍🏼